docs/static/pipeline-viewer.asciidoc

```diff
@@ -0,0 +1,107 @@
+[role="xpack"]
+[[logstash-pipeline-viewer]]
+=== Pipeline Viewer UI
+
+NOTE: The pipeline viewer is an {xpack} feature that requires a
+paid {xpack} license. See the
+https://www.elastic.co/subscriptions[Elastic Subscriptions] page for
+information about obtaining a license.
+
+//REVIEWERS: Not sure what license is required here. Does this come with basic?
+
+The pipeline viewer in {xpack} provides a simple way for you to visualize and
+monitor the behavior of complex Logstash pipeline configurations. Within the
+pipeline viewer, you can explore a directed acyclic graph (DAG) representation
+of the overall pipeline topology, data flow, and branching logic. The diagram
+is overlayed with important metrics, like events per second and time spent in
+milliseconds, for each plugin in the view.
+
+The diagram includes visual indicators to draw your attention to potential
+bottlenecks in the pipeline, making it easy for you to diagnose and fix
```

Comment:
I thought about including our formula, but isn't that likely to change? Is there some way we could describe the heuristic without being *too* precise?

Also, I wonder if this is something that will be configurable at some point. It should be...so users can change the setting that determines what gets flagged.